.circleci/config.yml

```diff
@@ -227,8 +304,55 @@ workflows:
           requires:
             - deci-common/persist_version_info
             - deci-common/codeartifact_login
+      
       - release_candidate: # happens on merge
           py_version: "3.7"
           requires:
             - "build3.7"
           <<: *release_candidate_filter
+
+  SG_docker:
+     jobs:
+       - build_and_publish_beta:
+           <<: *release_candidate_filter
+       - testing_supergradients_docker_image:
+          image_repo: '307629990626.dkr.ecr.us-east-1.amazonaws.com/deci/super-gradients'
+          image_tag: ${CIRCLE_TAG}-beta
```

Comment:
This is not how we create beta tag, please refer to a similar project like [deci-api](https://github.com/Deci-AI/deci-api/blob/master/.circleci/config.yml)

Reply:
on this case, you'd have to make a simple edit to replace `rc` with `b`. I;m pretty sure there is a command for that on our common orb, if there isn't please create one